packages/labs/task/README.md

```diff
@@ -2,85 +2,8 @@
 
 A controller for Lit that renders asynchronous tasks.
 
-## Overview
+This package has graduated from labs! It is now available as `@lit/task`. This package will no longer be updated, but it will continue to work and get updates through the ^1.0.0 version range of `@lit/task` because it's now just a re-export of that package. This should reduce duplication of code while everyone migrates their imports and dependencies.
```

Comment:
I think the "it's now just a re-export of that package" part should com earlier, then mention this package is a proxy and won't be updated.

We might want a headline here too.

Reply:
+1, done